pod2man.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
```

Comment:
No strong feelings, but if we're going to accommodate "set -e", then wonder if it's worth adding -u too (which could catch other issues), and then just drop all the question mark expansions.